root/components/common-macros.tt

```diff
@@ -834,9 +834,9 @@ END -%]
     };
     IF medium.name != '';
         args.medium_name = isolate_text(medium.name);
-        l('{medium_format} {position}: {medium_name}', args);
+        l('<span>{medium_format} {position}:</span> {medium_name}', args);
```

Comment:
There is no specific rule for this element in the style sheet as `> span` rules include `span.expand-triangle` also. Instead, I could wrap `{medium_name}` into a `span` with `medium-name` class to refer to in the style sheet in place of not-semantic `bdi`. Or do both?